paths_cli/wizard/volumes.py

```diff
@@ -0,0 +1,111 @@
+import operator
+from paths_cli.wizard.core import interpret_req
+from paths_cli.wizard.cvs import cvs
+
+def _vol_intro(wizard, as_state):
+    if as_state:
+        req = wizard.requirements['state']
+        if len(wizard.states) == 0:
+            intro = ("Now let's define stable states for your system. "
+                     f"You'll need to define {interpret_req(req)} of them.")
+        else:
+            intro = "Okay, let's define another stable state."
+    else:
+        intro = None
+    return intro
+
+def _binary_func_volume(wizard, op):
+    wizard.say("Let's make the first constituent volume:")
+    vol1 = volumes(wizard)
+    wizard.say(f"The first volume is:\n{str(vol1)}")
+    wizard.say("Let's make the second constituent volume:")
+    vol2 = volumes(wizard)
+    wizard.say(f"The second volume is:\n{str(vol2)}")
+    vol = op(vol1, vol2)
+    wizard.say(f"Created a volume:\n{str(vol)}")
```

Comment:
f-string auto `str` casting:
```suggestion
    wizard.say(f"Created a volume:\n{vol}")
```